.github/workflows/ci.yml

```
@@ -14,6 +14,7 @@ env:
   TD_QUEUE_ZONE: ${{ secrets.TD_QUEUE_ZONE }}
   DUMP_QUEUE_NAME: ${{ secrets.DUMP_QUEUE_NAME }}
   DUMP_QUEUE_ZONE: ${{ secrets.DUMP_QUEUE_ZONE }}
+  # If was necessary the variable GOPRIVATE can be set on the repository settings :: Secrets and Variables > action > Variables : GOPRIVATE
```

```suggestion
  # If needed the variable GOPRIVATE can be set in the repository settings :: Secrets and Variables > action > Variables : GOPRIVATE
```

I assume the value of the variable has to be what `go get` expects, i.e. the url of the private repo(s)?